client/sites/hosting/components/hosting-callout/index.tsx

```diff
@@ -15,13 +15,20 @@ export function HostingActivationCallout( {
 	siteId: number;
 	redirectUrl?: string;
 } ) {
-	const { data: siteTransferData } = useSiteTransferStatusQuery( siteId, {
-		refetchIntervalInBackground: true,
-	} );
+	const { data: siteTransferData, refetch: fetchSiteTransferStatus } = useSiteTransferStatusQuery(
+		siteId,
+		{
+			refetchIntervalInBackground: true,
+		}
+	);
 
 	const isActivating = siteTransferData?.isTransferring;
 	const isActivated = transferStates.COMPLETED;
```

Comment:
This is currently set to the 'completed' string, which always returns true. Shouldn't this be set from `siteTransferData?.isTransferring` instead?

Reply:
Oops...it was wrong 😂